SU2_CFD/include/variables/CIncEulerVariable.hpp

```diff
@@ -101,7 +99,19 @@ class CIncEulerVariable : public CFlowVariable {
     Primitive(iPoint, indices.Density()) = val_density;
     return val_density <= 0.0;
   }
+  
+  inline void Set_Density_time_n(unsigned long iPoint, su2double val) {
+    Density_time_n[iPoint] = val;
+  }
 
+  inline void Set_Density_unsteady(unsigned long iPoint, su2double val) {
+    Density_unsteady[iPoint] = val;
+  }
```

Comment:
```suggestion
  inline void SetDensity_Unsteady(unsigned long iPoint, su2double val) {
    Density_unsteady[iPoint] = val;
  }
```
to keep consistency with the other function names, like **S**et**V**elocity_**O**ld